server_admin/topics/clients/oidc/con-confidential-client-credentials.adoc

```diff
@@ -32,19 +32,19 @@ image:{project_images}/generate-client-keys.png[]
 . Enter a store password.
 . Click *Generate and Download*.
 
-When you generate the keys, {project_name} will store the certificate and you must download the private key and certificate for your client.
+When you generate the keys, {project_name} will store the certificate and you download the private key and certificate for your client.
 
-You can also generate keys using an external tool and then import the client's certificate by clicking the *Import Certificate* button.
+You can also generate keys using an external tool and then import the client's certificate by clicking *Import Certificate*.
 
 .Import Certificate
 image:{project_images}/import-client-cert.png[]
 
 . Select the archive format of the certificate.
 . Enter the store password.
-. Select the certificate file by clicking the *Import File* button.
+. Select the certificate file by clicking *Import File*.
 . Click *Import*.
 
-It is not necessary to import a certificate if you click the *Use JWKS URL* switch. In this case, you can provide the URL where the public key is published in https://self-issued.info/docs/draft-ietf-jose-json-web-key.html[JWK] format. With this option, if the key is ever changed, {project_name} reimports the key.
+It is not necessary to import a certificate if you click *Use JWKS URL*. In this case, you can provide the URL where the public key is published in https://self-issued.info/docs/draft-ietf-jose-json-web-key.html[JWK] format. With this option, if the key is ever changed, {project_name} reimports the key.
```

Comment:
I also try to avoid "It is.."  Again what is the subject of the sentence. 
"Importing a certificate is unnecessary if you click *Use JWKS URL*."